docs/content/action/config.md

```diff
@@ -3,72 +3,67 @@ title: Config
 sort: 1
 ---
 
-# Metadata "config"
+# Configuration properties
 
-Config are used to contextualise the handler function.
+Configuration properties are used to contextualize the handler function.
```

Comment:
What is the difference between a metadata and an config, config are properties tied an action while metadata influence the behavior of multiple actions, we shall see this. You description is to generic